packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -854,7 +854,32 @@ const instance = new rds.DatabaseInstance(this, 'Instance', {
 });
 ```
 
-**Note**: In addition to the setup above, you need to make sure that the database instance has network connectivity
+The following example shows enabling domain support for a Aurora databse cluster and creating an IAM role to access Directory Services.
```

Comment:
Since there is a similar sentence before, it is good to keep this one simple.

```suggestion
You can also use the Kerberos authentication for an Aurora database cluster.
```